### PR TITLE
Column layout

### PR DESCRIPTION
**Pull request by BRHoward**
Dynamic height for tweet objects based on amount of tweets in a column and the size of the window.
Removed tweet column directive for now just for simplicity.

### CODE REVIEW

**Review by stephen-lt-sl — APPROVED** [review]

File: client/angular/controllers/MainController.js [hunk 1]
```diff
@@ -8,35 +8,59 @@
...
+        //defines the space between all tweets on the wall
+        var tweetMargin = 12;
+
         $scope.tweets = [];
+
         vm.updates = [];
 
+        var shouldBeDisplayed = function(tweet) {
+            var display = $scope.loggedIn || !((tweet.blocked && !tweet.display) || tweet.deleted || tweet.hide_retweet);
+            console.log(display);
```
Comment (susanthomson) [inline client/angular/controllers/MainController.js]: :baby_symbol:

File: client/templates/tweet-column.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<div layout="column" layout-fill">
```
Comment (susanthomson) [inline client/templates/tweet-column.html]: There's an extra " here that's upsetting my text editor

File: client/css/main.css [hunk 3]
```diff
@@ -27,7 +23,9 @@ html::-webkit-scrollbar {
 }
 
 .banner {
```
Comment (susanthomson) [inline client/css/main.css]: I don't think we're even using this?

**Review by susanthomson — APPROVED** [review]
:dizzy:

**Pull request merged**